### ISSUE TITLE
04: visualize many panicle

### ISSUE THREAD

**Issue by heleneadam** [othomantegazza/ptrapr#6]
don't undertstant this : 
# Store file names in named object, to use them later
panicle_paths <-
  panicle_paths %>% 
  {set_names(x = paste(path_to_folder, ., sep = "/"), nm = .)}

could you give me a stupid example..:)

**Comment by othomantegazza** [comment]
Yes, that's a bit condensed, I'll add more comments and explanations :)

**Comment by othomantegazza** [comment]
Sorry, it took me a while.

At the beginning there was a passage that was unclear: you have to store the path to your folder in a variable called path_to_folder.

I improved a bit how the first steps of vignette 4 are written, please, try to see if it works and let me know.


Best :)

**Comment by othomantegazza** [comment]
https://othomantegazza.github.io/ptrapr/articles/04-visualize-many-panicles-loop-r.html

**Comment by heleneadam** [comment]
hi otho

It is more clear..and everything works perfectly….

Hélène Adam

UMR DIADE
EDI Group
IRD (Institut de Recherche pour le Développement)
BP64501
911, avenue Agropolis
F-34394 Montpellier Cedex 5
FRANCE
tel: +33 4 67 41 62 41
e-mail: helene.adam@ird.fr <mailto:helene.adam@ird.fr>
